src/Longrunning/OperationsClient.php

```diff
@@ -121,17 +117,16 @@ private static function getPageStreamingDescriptors()
      * Constructor.
      *
      * @param array $options {
-     *                       Optional. Options for configuring the service API wrapper.
+     *                       Required. Options for configuring the service API wrapper. Those options
+     *                       that must be provided are marked as Required.
      *
-     *     @type string $serviceAddress The domain name of the API remote host.
-     *                                  Default 'longrunning.googleapis.com'.
+     *     @type string $serviceAddress Required. The domain name of the API remote host.
```

Comment:
If this is required, should it be a proper parameter instead of an array parameter? (I'll let others decide, but I wanted to ask)